services/grafana/provisioning/datasources/datasource.yaml

```diff
@@ -4,17 +4,17 @@ apiVersion: 1
 # what's available in the database
 datasources:
   # <string, required> name of the datasource. Required
-- name: InfluxDB
+- name: UAV
   # <string, required> datasource type. Required
   type: influxdb
   # <string, required> access mode. proxy or direct (Server or Browser in the UI). Required
   access: proxy
   # <int> org id. will default to orgId 1 if not specified
   orgId: 1
   # <string> url
-  url: http://${INFLUX_HOST}:${INFLUX_PORT}
+  url: http://influx:8086
   # <string> database name, if used
-  database: ${DB_NAME}
+  database: lumberjack
```

Comment:
Hmm, I don't recommend changing this. It makes it more difficult to change hostnames and ports for alternate deployments. That is why we used the [`uavaustin/grafana`](https://github.com/uavaustin/orchestra/blob/logging-service/services/grafana/Dockerfile) image instead of the default `grafana/grafana` image.

Reply:
I couldn't get the custom `uavaustin/grafana` image working. But for environment variable interpolation, [base grafana allows us to do this](https://github.com/grafana/grafana/pull/16499). In the latest commit, I added back the environment variables in data sources.

Reply:
I couldn't get the custom uavaustin/grafana image working. But for environment variable interpolation, base grafana allows us to do this. In the latest commit, I added back the environment variables in data sources.